tests/ikev1-rules/test.rules

```diff
@@ -0,0 +1,18 @@
+alert ikev1 any any -> any any (msg:"ikev1 initiator"; ikev1.spi_initiator; pcre:"/.*/"; sid:1;)
```

Comment:
This is a strange signature matching for any initiator. Can we be more specific ?